### PR TITLE
Psyf's fixes for v1.4.1

### PR DESCRIPTION
**Pull request by Psyf**
Please take a look at the referenced issues for details on this small PR.

### CODE REVIEW

**Comment by Psyf** [comment]
Ready for Review and Merging @trufflepirate @teojunjie @Ftywan

File: src/main/java/seedu/address/ui/job/JobListPanel.java [hunk 1]
```diff
@@ -30,6 +32,19 @@ public JobListPanel(ObservableList<Job> jobList) {
         super(FXML);
         setConnections(jobList);
         registerAsAnEventHandler(this);
+
```
Comment (teojunjie) [inline src/main/java/seedu/address/ui/job/JobListPanel.java]: Sorry i may have missed out something, but i assume this is for your auto job addition function?

File: src/main/java/seedu/address/logic/commands/job/AddJobCommand.java [hunk 2]
```diff
@@ -57,6 +58,10 @@ public CommandResult execute(Model model, CommandHistory history) throws Command
         }
 
         try {
+            //TODO: this is breaking after one auto addition for some reason
```
[the remark this answers is not in the document]
Comment (Psyf) [reply src/main/java/seedu/address/logic/commands/job/AddJobCommand.java]: Oh AUTO was re-enabled, but //TODO not removed. 
Thanks for pointing it out.

File: src/main/java/seedu/address/logic/commands/job/ManageJobCommand.java [hunk 3]
```diff
@@ -66,17 +66,14 @@ public CommandResult execute(Model model, CommandHistory history) throws Command
             model.updateFilteredMachineList(PREDICATE_SHOW_ALL_MACHINES);
             return new CommandResult(MESSAGE_CANCELLED_JOB);
         } else if (option.equals(OPTION_DELETE)) {
```
[the remark this answers is not in the document]
Comment (Psyf) [reply src/main/java/seedu/address/logic/commands/job/ManageJobCommand.java]: Wait, what is empty?

File: src/main/java/seedu/address/ui/job/JobListPanel.java (hunk 1 above)
Comment (Psyf) [reply src/main/java/seedu/address/ui/job/JobListPanel.java]: Oh no, this is autoRefreshing now. You don't have to click on something else before JobStatus changes or anything.

**Comment by trufflepirate** [comment]
Reviewed, looks good. Merging.

**Pull request merged**